examples/fabric-bridge-app/linux/BUILD.gn

```diff
@@ -33,10 +33,11 @@ if (bridge_enable_pw_rpc) {
 executable("fabric-bridge-app") {
   sources = [
     "${chip_root}/examples/fabric-bridge-app/fabric-bridge-common/include/CHIPProjectAppConfig.h",
-    "Device.cpp",
-    "DeviceManager.cpp",
-    "include/Device.h",
-    "include/DeviceManager.h",
+    "${chip_root}/examples/fabric-bridge-app/fabric-bridge-common/include/Device.h",
```

Comment:
Wouldn't it be better to create a BUILD.gn file for "fabric-bridge-common" similar to the code below?
https://github.com/project-chip/connectedhomeip/blob/master/examples/lighting-app/lighting-common/BUILD.gn

Reply:
Acked